<launch>
    <arg name="model_name" default="msj_platform"/>
    <param name="robot_description" textfile="$(find robots)/$(arg model_name)/model.urdf" />
    <rosparam file="$(find robots)/$(arg model_name)/config/controller.yaml" command="load"/>
    <param name="Kp" type="double" value="100"  />
    <param name="Kd" type="double" value="10"  />
    <rosparam param="q_target">[0.0, 0.0, 0.0]</rosparam>
    <param name="min_force" type="double" value="0"  />
    <param name="max_force" type="double" value="500"  />
    <node name="controller_manager"
          pkg="controller_manager"
          type="spawner"
          args="sphere_axis0 sphere_axis1 sphere_axis2" />

    <node name="robot_controller"
          pkg="kindyn"
          type="test_robot"
          respawn="false"
          output="screen"
          args="$(find robots)/$(arg model_name)/model.urdf $(find robots)/$(arg model_name)/cardsflow.xml"
    />

</launch>
        <!--launch-prefix="gdb -ex run"-->
        <!--launch-prefix="gdb -ex run"-->
        <!-- FOR DEBUGGING USE THIS launch-prefix="xterm -e gdb -se devel/lib/myo_master/multiJoint" FOR VALGRIND USE THIS launch-prefix="valgrind -log-file=/home/roboy/workspace/ros_hierarchy/valgrind.log -leak-check=full"-->

        <!-- args="-c /home/roboy/workspace/roboy-ros-control/src/roboy_powerlink/powerlink/output/mnobd.cdc"-->